### ISSUE TITLE
aquifer.local.json doesn't full overload run-API scripts

### ISSUE THREAD

**Issue by generalredneck** [aquifer/aquifer#177]
in my main aquifer.json I have

```
"run": {
  "scripts": {
    "hello": [
        "echo 1",
        "echo 2",
        "echo 3",
        "echo 4",
        "echo 5"
      ]
    }
  }
```

In my local I have

```
"run": {
  "scripts": {
      "hello": [
        "echo A",
        "echo B",
        "echo C"
      ]
    }
}
```

my output is

```
> aquifer "run" "hello"

A
B
C
4
5
```

**Comment by chasingmaxwell** [comment]
This is addressed by #178

**Comment by chasingmaxwell** [comment]
PR was merged
